scripts/build/install-build-pre-requisites.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
```

Comment:
This file doesnt exists any more looks like this is merge issue. can you please revert this file?